log10/load.py

```diff
@@ -42,9 +45,8 @@ def wrapper(*args, **kwargs):
             res = requests.request("POST",
                                    session_url + "/" + completionID, headers={"x-log10-token": token, "Content-Type": "application/json"}, json={
                                        "response": json.dumps(output),
+                                       "status": "finished",
                                        "duration": int(duration),
-                                       "orig_module": func.__module__,
```

Comment:
just making sure - you don't need these in the completion request?

Reply:
It is an upset on the same row, so wouldn't be necessary in this one